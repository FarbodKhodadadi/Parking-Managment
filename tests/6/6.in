request_spot carD
assign_spot 44
assign_spot 22
request_spot carC
request_spot carA
request_spot carB
pass_time 6
checkout 44
request_spot carD
pass_time 2
checkout 22
