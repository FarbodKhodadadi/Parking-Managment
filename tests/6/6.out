11: normal 455 23
44: covered 505 53
Spot 44 is occupied now.
Spot 22 is occupied now.
33: covered 395 108
11: normal 455 23
55: normal 700 56
Spot 44 is free now.
Total cost: 823
11: normal 455 23
44: covered 505 53
Spot 22 is free now.
Total cost: 1708
